### PR TITLE
Fix _UI

### PR DESCRIPTION
**Pull request by Smokashi23**
Hide the EditIcon from employee where he cant access it
Hide the fields from employee where he can't edit it

### CODE REVIEW

File: src/modules/details/components/EditTicketForm.tsx [hunk 1]
```diff
@@ -229,6 +229,7 @@ export const EditTicketForm = ({
             onChange={handleChange}
             error={touched.status && Boolean(errors.status)}
           >
+          {userAuth.role !== 'employee' && (
```
Comment (Chetan-Satpute-Josh) [inline src/modules/details/components/EditTicketForm.tsx:232]: please create enums for user auth role and ticket status

File: src/modules/details/components/EditTicketForm.tsx [hunk 2]
```diff
@@ -237,6 +238,7 @@ export const EditTicketForm = ({
             >
               <span>{ticket?.status}</span>
             </MenuItem>
+          )}
             {ticket?.permited_transitions?.map((item) => (
```
Comment (Chetan-Satpute-Josh) [inline src/modules/details/components/EditTicketForm.tsx:242]: map items in component body

File: src/modules/details/components/EditTicketForm.tsx (hunk 1 above)
Comment (Smokashi23) [reply src/modules/details/components/EditTicketForm.tsx:232]: in codebase they have used ROLES constant can i use that if i change constant to enum i need to change everywhere in code where they have used ROLES constant

Comment (Chetan-Satpute-Josh) [reply src/modules/details/components/EditTicketForm.tsx:232]: Yes use the constant

**Review by Chetan-Satpute-Josh — APPROVED** [review]

**Pull request merged**